src/editor/index.tsx

```diff
@@ -48,8 +49,9 @@ interface EditorState {
 }
 
 class Editor extends React.Component<EditorProps, EditorState> {
-  static defaultProps = {
+  static defaultProps: Partial<EditorProps> = {
     value: '',
+    editorWrapStyle: 'hard',
```

Comment:
In MDN, 'soft' is the default, but in the current version 'hard 'was the initial value, so 'hard' is set to the initial value.